File: app/src/main/java/com/example/test/aac/room/UserProfile.java
package com.example.test.aac.room;

import androidx.room.Entity;
import androidx.room.PrimaryKey;

@Entity
public class UserProfile {
    @PrimaryKey(autoGenerate = true)
    public int id;
    public String name;
    public String address;
    public String phone;
}

File: app/src/main/java/com/example/test/aac/data/UserProfile.kt
package com.example.test.aac.data

data class UserProfile(
        val n:String = "없음",
        val a:String = "없음",
        val p:String = "000-0000-0000",
        val g:Boolean = true
) {
    var name: String = n
    var address:String = a
    var phone: String = p
    var gender: Boolean = g
}